### PR TITLE
chore: add CODEOWNERS file

### LINKED ISSUE #139: Add Repo Maintainers to CODEOWNERS file
# Update CODEOWNERS to provide maintainers ownership of all files in their repos

**Description**:

Update this repositories `.github/CODEOWNER` file to ensure that the repository maintainer
group(s) have full control of the files in this repository.

### PR DESCRIPTION
**Pull request by PavelSBorisov**
**Description**:
Update CODEOWNERS to provide maintainers ownership of all files in their repos

**Related issue(s)**:

Fixes #139 

**Notes for reviewer**:
N/A

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by andrewb1269hg — APPROVED** [review]
Looks good.

**Review by andrewb1269hg — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,29 +1,29 @@
 # Default code owners for entire repository
-*                                               @hashgraph/transaction-tool @hashgraph/hedera-base @hashgraph/marketing
+*                                               @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/platform-base @hashgraph/hedera-services
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:2]:
```suggestion
*                                               @hashgraph/transaction-tool @hashgraph/hedera-services @hashgraph/marketing
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
 # Protection Rules for Github Configuration Files and Actions Workflows
-/.github/                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers
+/.github/                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/platform-base @hashgraph/hedera-services
 
 # Codacy Tool Configurations
-/config/                                        @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/transaction-tool @hashgraph/hedera-base
-.remarkrc                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/transaction-tool @hashgraph/hedera-base
+/config/                                        @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services
+.remarkrc                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:14]:
```suggestion
/config/                                        @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services  @hashgraph/transaction-tool
.remarkrc                                       @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services  @hashgraph/transaction-tool
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
 
 # Self-protection for root CODEOWNERS files (this file should not exist and should definitely require approval)
 /CODEOWNERS                                     @hashgraph/release-engineering-managers
 
 # Protect the repository root files
-/README.md                                      @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/transaction-tool @hashgraph/hedera-base
+/README.md                                      @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:20]:
```suggestion
/README.md                                      @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services  @hashgraph/transaction-tool
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
 **/LICENSE                                      @hashgraph/release-engineering-managers
 
 # CodeCov configuration
-**/codecov.yml                                  @hashgraph/devops-ci @hashgraph/release-engineering-managers
+**/codecov.yml                                  @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/platform-base @hashgraph/hedera-services
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:24]:
```suggestion
**/codecov.yml                                  @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/platform-base @hashgraph/hedera-services @hashgraph/transaction-tool
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,29 +1,29 @@
...
 
 # Git Ignore definitions
-**/.gitignore                                   @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers
-**/.gitignore.*                                 @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers
+**/.gitignore                                   @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services
+**/.gitignore.*                                 @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services
```
Comment (rbarkerSL) [inline .github/CODEOWNERS:28]:
```suggestion
**/.gitignore                                   @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services @hashgraph/transaction-tool
**/.gitignore.*                                 @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/devops-ci-committers @hashgraph/platform-base @hashgraph/hedera-services @hashgraph/transaction-tool
```

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -1,29 +1,29 @@
 # Default code owners for entire repository
```
Comment (PavelSBorisov) [inline .github/CODEOWNERS:1]: @rbarkerSL, I've committed your suggestions but I also noticed that the @hashgraph/platform-base and @hashgraph/hedera-services teams don't seem to be valid here according to GitHub. Could you please take a look?

Comment (andrewb1269hg) [reply .github/CODEOWNERS:1]: @PavelSBorisov I have resolved this issue by adding the two groups to the repository access.

**Review by rbarkerSL — APPROVED** [review]

**Pull request merged**